test/evergreen.yml

```diff
@@ -1637,6 +1661,111 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: long-test-1
```

Comment:
Though it was included as part of the long test job in Jenkins before, I think we should give it a more meaningful name to describe what type of tests to be executed e.g. `ftruncate-test`, as it's being split into a separate task.

Reply:
Good idea. I hav renamed the tests accordingly.